### DISCUSSION TITLE
How to change Pydantic default __root__ example name

### DISCUSSION THREAD

**Discussion by DEADSEC-SECURITY** [pydantic/pydantic#4721] [Question] [answered]
So I have this class:
```
class Table(BaseModel):
    __root__: Dict[int, Test]
```
and I'm using the __root__ since it is a dynamic value but when I go to /redoc (I'm using FastAPI) the example values it defaults to is property1, property2 as in the image below but I wanted the default example to be id_1, id_2 for example. How can I achieve this?
![D9W5b](https://user-images.githubusercontent.com/45077772/200036432-dceea063-5bf6-4ee4-a13e-72450b2c92e3.png)
I tried changing the alias and title using the Field method of the library but didn't seem to work like I expected.
I also tried using `schema_extras` but the problem with it is that I would have to define the hole example and in my setup I have multiple classes since this is a rather complex object and making examples manually for everything would be kinda messy.

**Comment by RonRothman** [comment, marked as answer]
Looks like your question [has been answered here](https://stackoverflow.com/questions/74319416/how-to-change-pydantic-default-root-example).
